src/main/resources/liquibase/workbench_changelog/12_5_0.xml

```diff
@@ -0,0 +1,713 @@
+				<columnExists tableName="users_roles" columnName="created_by"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add created_by
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN created_by INT(11) NULL;
+
+			ALTER TABLE users_roles
+			ADD INDEX fk_users_roles_2_idx (created_by ASC);
+			ALTER TABLE users_roles
+			ADD CONSTRAINT fk_users_roles_2
+			FOREIGN KEY (created_by)
+			REFERENCES users (userid)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users_roles" columnName="created_date"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add created_date
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN created_date TIMESTAMP NULL DEFAULT NULL;
+
+			alter table users_roles
+			  modify COLUMN created_date TIMESTAMP NULL default CURRENT_TIMESTAMP;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="role_type"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create table role_type
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE role_type (
+			role_type_id INT(11) NOT NULL,
+			name VARCHAR(100) NOT NULL,
+			PRIMARY KEY (role_type_id));
+
+			INSERT INTO role_type VALUES (1, "INSTANCE");
+			INSERT INTO role_type VALUES (2, "CROP");
+			INSERT INTO role_type VALUES (3, "PROGRAM");
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-6">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<columnExists tableName="role" columnName="description"/>
+				<not>
+					<columnExists tableName="role" columnName="name"/>
+				</not>
+			</and>
+		</preConditions>
+		<comment>
+			Alter table role to changing name by description
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE role
+			CHANGE COLUMN description name VARCHAR(100) NOT NULL;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-7">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="role" columnName="description"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table role to add description
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE role
+			ADD COLUMN description VARCHAR(255) NULL AFTER NAME;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-8">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="role" columnName="role_type_id"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table role to add role_type_id
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE role
+			ADD COLUMN role_type_id INT(11) NULL AFTER DESCRIPTION;
+
+			update role set role_type_id = 1 where id = 1;
```

Comment:
It would be good to provide comments here for each shipped role and the role type assigned.